lib/sedna/algorithms/multi_task_learning/task_jobs/artifact.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+__all__ = ('Task', 'TaskGroup', 'Model')
+
+
+class Task:
+    def __init__(self, entry, samples, meta_attr=None):
+        self.entry = entry
+        self.samples = samples
+        self.meta_attr = meta_attr
+        self.model = None  # define on running
```

Comment:
Using word assignment is better, since you have already defined `self.model`

Reply:
fix